streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -609,6 +609,7 @@ private KafkaStreams createKafkaStreams(String topic, final CountDownLatch latch
 
                     @Override
                     public void init(ProcessorContext context) {
+                        super.init(context);
```

Comment:
might be better to remove the overwrite method completely?

Reply:
Alternatively, if we don't care about the context, we could just use `Processor` directly instead of `AbstractProcessor`.

Reply:
Sounds good to me.